x-pack/plugins/index_lifecycle_management/__jest__/client_integration/helpers/actions/toggle_phase_action.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { TestBed } from '@kbn/test/jest';
+
+import { act } from 'react-dom/test-utils';
+import { Phase } from '../types';
+
+const toggleDeletePhase = async (testBed: TestBed) => {
+  const { find, component } = testBed;
+
+  let button = find('disableDeletePhaseButton');
+  if (!button.length) {
+    button = find('enableDeletePhaseButton');
```

Comment:
A button to disable the delete phase is only shown, when the delete phase is enabled.